README.md

```diff
@@ -184,6 +184,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Anshika Saini](https://github.com/Anshikaa-Saini)
  -  [Anushree mehta](https://github.com/anushreemehta6)
   - [Aryan Karamtoth](https://github.com/spaciouscoder78)
+  -  [Aniket Upadhyay](https://github.com/upadhyayaniket29)
```

Comment:
Make my name

Reply:
I didn't  get you @upadhyayaniket29 
mention your query more elaborately